docs/pages/connect-your-client/teleport-connect.mdx

```diff
@@ -158,27 +158,74 @@ installation directory to the `Path` user environment variable.
 </TabItem>
 </Tabs>
 
+## Configuration
+
+Teleport Connect can be configured by editing the `app_config.json` file.
```

Comment:
```suggestion
Teleport Connect can be configured by editing the `app_config.json` file, which it creates on first installation.
```

Reply:
Changed to `first launch` to be more precise.